src/cfg.rs

```diff
@@ -25,6 +28,60 @@ pub struct Config {
     pub colors: [Color; 256]
 }
 
+impl Debug for Config {
+    fn fmt(&self, f: &mut Formatter) -> Result {
+        write!(f, "Config {{ }}")
+    }
+}
+impl PartialEq for Config {
+    fn eq(&self, other: &Config) -> bool {
+        if self.font != other.font {
+            return false;
+        }
+        if self.scrollback != other.scrollback {
+            return false;
+        }
+        if self.tab_stop != other.tab_stop {
+            return false;
+        }
+        if self.fg_color != other.fg_color {
+            return false;
+        }
+        if self.bg_color != other.bg_color {
+            return false;
+        }
+        if self.cursor_color != other.cursor_color {
+            return false;
+        }
+        for i in 0..255 {
+            if self.colors[i] != other.colors[i] {
+                return false;
+            }
+        }
+        true
+    }
+}
+
+impl Clone for Config {
+    fn clone(&self) -> Config {
+        Config{
+            font: self.font.clone(),
+            scrollback: self.scrollback.clone(),
+            tab_stop: self.tab_stop.clone(),
+            fg_color: self.fg_color.clone(),
+            bg_color: self.bg_color.clone(),
+            cursor_color: self.cursor_color.clone(),
+            colors: self.colors,
```

Comment:
Hah, I was wondering why you had a manual impl for `Clone`. I can't wait until Rust has const generics and we don't have to do silly things because arrays with lengths above 32 don't implement anything.